.github/workflows/ci_rust.yml

```diff
@@ -150,6 +157,36 @@ jobs:
         working-directory: ${{env.ROOT_PATH}}
         run: cargo test --all-features
 
+  fips:
```

Comment:
Yea, it is but I wanted an explicit (rather than implicit via --all-features) test for fips since it results in a totally different binary and compilation process.